openscenario/openscenario_interpreter/include/openscenario_interpreter/syntax/relative_object_position.hpp

```diff
@@ -0,0 +1,62 @@
+// Copyright 2015 TIER IV, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef OPENSCENARIO_INTERPRETER__SYNTAX__RELATIVE_OBJECT_POSITION_HPP_
+#define OPENSCENARIO_INTERPRETER__SYNTAX__RELATIVE_OBJECT_POSITION_HPP_
+
+#include <openscenario_interpreter/scope.hpp>
+#include <openscenario_interpreter/simulator_core.hpp>
+#include <openscenario_interpreter/syntax/double.hpp>
+#include <openscenario_interpreter/syntax/entity_ref.hpp>
+#include <openscenario_interpreter/syntax/orientation.hpp>
+#include <pugixml.hpp>
+#include <traffic_simulator_msgs/msg/lanelet_pose.hpp>
+
+namespace openscenario_interpreter
+{
+inline namespace syntax
+{
+/* ---- RelativeObjectPosition --------------------------------------------------
```

Comment:
Please put OpenSCENARIO version after element name like below
https://github.com/tier4/scenario_simulator_v2/blob/master/openscenario/openscenario_interpreter/include/openscenario_interpreter/syntax/deterministic_multi_parameter_distribution.hpp#L26

Reply:
fixed in 
[fb34fc1](https://github.com/tier4/scenario_simulator_v2/pull/1030/commits/fb34fc1db23e879f1b8d3426acadf8c31b9f17cb)